### DISCUSSION TITLE
About the use of punctuation marks in Chinese

### DISCUSSION THREAD

**Discussion by roc916** [abpframework/abp#12959] [Ideas]
I see many Chinese documents and localization files commonly use English punctuation.

e.g. [Volo.Abp.ExceptionHandling/*/zh-Hans.json#L4](https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.ExceptionHandling/Volo/Abp/ExceptionHandling/Localization/zh-Hans.json#L4), [Startup-Templates/Index.md](https://github.com/abpframework/abp/blob/dev/docs/zh-Hans/Startup-Templates/Index.md)

I think punctuation should be used in accordance with local language writing conventions.

especially for localization files.

**Comment by maliming** [comment]
It's not that important. If necessary, you can replace it yourself. Different people have different needs.
